pkgs/development/python-modules/pandas/default.nix

```diff
@@ -227,6 +227,9 @@ buildPythonPackage rec {
     "test_binops"
     # These tests are unreliable on aarch64-darwin. See https://github.com/pandas-dev/pandas/issues/38921.
     "test_rolling"
+  ] ++ lib.optional stdenv.is32bit [
+    # https://github.com/pandas-dev/pandas/issues/55789
```

Comment:
Could this be updated to `https://github.com/pandas-dev/pandas/issues/37398`? I'm making tooling to scan for closed issues with fixes tagged in a release

Reply:
Done!